app/presenters/nav_tab/link_collection.rb

```diff
@@ -27,15 +27,24 @@ def self.tab_methods
   end
 
   def admin
-    default + admin_only
+    use_button + admin_only
   end
 
   def customer
-    default + [orders, reservations, payment_sources, files]
+    default
   end
 
   def default
-    [home]
+    [orders, reservations, payment_sources, files]
+  end
+
+  def use_button
+    [use] + [manage]
+  end
+
+  def home_button
+    return use_button if SettingsHelper.feature_on?(:use_manage)
```

Comment:
Can we do this as an if/else? If we do the guard clause here then the `return` on the following line is not necessary.